### PR TITLE
How to get a list of application channels with subscription counts

### PR DESCRIPTION
**Pull request by jameshfisher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -241,6 +241,21 @@ This can also be achieved using the generic `pusher->get` function:
 $pusher->get( '/channels', array( 'filter_by_prefix' => 'presence-' ) );
 ```
 
+### Get a list of application channels with subscription counts
+
+The REST API returning the channel list does not support returning the subscription count along with each channel. Instead, you can fetch this data by iterating over each channel and making another request:
```
Comment (WillSewell) [inline README.md]: We call it the `HTTP API`, not `REST API`.

**Comment by mdpye** [comment]
Enumerating seems fine from an expense PoV, as each request is a billable message. See internal chat for a discussion of the API endpoint implementation itself.

**Comment by WillSewell** [comment]
Good point. It is potentially something that could be added to libraries then with a warning: "this consumes (number of channels + 1) messages!"

**Comment by WillSewell** [comment]
Merging because even if this could be useful elsewhere too, it's better than nothing having it here!

**Pull request merged**